client/state/stored-cards/reducer.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * External dependencies
+ */
+import { combineReducers } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+import {
+	STORED_CARDS_FETCH,
+	STORED_CARDS_FETCH_COMPLETED,
+	STORED_CARDS_FETCH_FAILED,
+	STORED_CARDS_DELETE,
+	STORED_CARDS_DELETE_COMPLETED,
+	STORED_CARDS_DELETE_FAILED
+} from 'state/action-types';
+
+import { createStoredCardsArray } from './assembler.js';
+
+/**
+ * List all known stored cards of the current user at /me/stored-cards.
```

Comment:
So yes we should probably avoid mentioning the `/me/stored-cards` API endpoint since it doesn't really matter where that data is fetched from - it's a different concern.

To answer your question I found it confusing for several reasons:
* This function doesn't only return a list of cards but also flags
* This function also takes care of deleting cards
* The docblock comment of this function is not in line with the ones of other reducer functions below:
```
`Reducer` function which handles request/response actions concerning stored cards fetching
```